packages/react/src/components/loaded/loaded.tsx

```diff
@@ -0,0 +1,32 @@
+import { SDKState } from "../../context/slash-id-context";
+import { useSlashID } from "../../main";
+
+interface Props {
+    fallback?: React.ReactNode
+    children: React.ReactNode
+}
+
+/**
+ * Conditional rendering helper
+ * 
+ * Acts as a guard for SlashID core SDK dependent operations
+ */
+export const SlashIDLoaded: React.FC<Props>  = ({ fallback, children }) => {
+    const { sdkState } = useSlashID()
+
+    if (sdkState !== SDKState.Ready) {
```

Comment:
Not including `authenticating` here might have side effects - whenever `logIn` is executed, state switches from `ready` to `authenticating`.

When I was working on that initially I figured it would make sense but now I'm having second thoughts as I cannot even think of a use case for that - authentication states are handled inside the `<Form>`.

So IMO the solution would be to actually remove the `authenticating` state from the context given that it does not really correspond to the lifecycle of the SDK. What do you think @jake-slashid? Of course we should look for any side effects of that if we do remove it.

Reply:
Agreed then! 👍 Let's remove it 😄

Reply:
It's gone 🪄 ✨